blocks/data-list/data-list.js

```diff
@@ -0,0 +1,112 @@
+import { fetchPlaceholders } from '../../scripts/aem.js';
+
+// Fetch placeholders
+const placeholders = await fetchPlaceholders('');
+
+const {
+  sNo, countries, continent, capital, abbreviation,
```

Comment:
Keep the standard naming convention for block specific placeholders.

Ex: BLOCKNAME-KEY

Reply:
done